guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -149,6 +175,10 @@ ifdef::satellite[]
 ----
 endif::[]
 
+ifdef::foreman-el,katello[]
+include::proc_configuring-repositories-el.adoc[]
+endif::[]
```

Comment:
this was removed in https://github.com/theforeman/foreman-documentation/pull/1279/ even after I explicitly said it is required in https://github.com/theforeman/foreman-documentation/pull/1279/files#r885316255

Reply:
I am sorry for this but it was breaking the build. I have replied to the same in #1279 (https://github.com/theforeman/foreman-documentation/pull/1279#discussion_r890961433).

Reply:
It was, yes, and the correct solution was to fix the build instead of removing it.